SPECS/objectweb-asm/objectweb-asm.spec

```diff
@@ -14,136 +14,96 @@
 
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
+%global debug_package %{nil}
 
 Name:           objectweb-asm
-Version:        7.2
-Release:        5%{?dist}
+Version:        9.6
+Release:        1%{?dist}
 Summary:        Java bytecode manipulation framework
 License:        BSD-3-Clause
 Group:          Development/Libraries/Java
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://asm.ow2.io/
 # ./generate-tarball.sh
-Source0:        %{_distro_sources_url}/%{name}-%{version}.tar.xz
-Source1:        %{name}-%{version}-build.tar.xz
-Source2:        http://repo1.maven.org/maven2/org/ow2/asm/asm/%{version}/asm-%{version}.pom
-Source3:        http://repo1.maven.org/maven2/org/ow2/asm/asm-analysis/%{version}/asm-analysis-%{version}.pom
-Source4:        http://repo1.maven.org/maven2/org/ow2/asm/asm-commons/%{version}/asm-commons-%{version}.pom
-Source5:        http://repo1.maven.org/maven2/org/ow2/asm/asm-test/%{version}/asm-test-%{version}.pom
-Source6:        http://repo1.maven.org/maven2/org/ow2/asm/asm-tree/%{version}/asm-tree-%{version}.pom
-Source7:        http://repo1.maven.org/maven2/org/ow2/asm/asm-util/%{version}/asm-util-%{version}.pom
-# We still want to create an "all" uberjar, so this is a custom pom to generate it
-# TODO: Fix other packages to no longer depend on "asm-all" so we can drop this
-Source9:        asm-all.pom
-# The source contains binary jars that cannot be verified for licensing and could be proprietary
-Source10:       generate-tarball.sh
-BuildRequires:  ant
-BuildRequires:  fdupes
-BuildRequires:  java-devel >= 1.8
+Source0:        %{_distro_sources_url}/%{name}-%{version}.tar.gz
+Source1:        aggregator.pom
+Source2:        https://repo1.maven.org/maven2/org/ow2/asm/asm/%{version}/asm-%{version}.pom
+Source3:        https://repo1.maven.org/maven2/org/ow2/asm/asm-analysis/%{version}/asm-analysis-%{version}.pom
+Source4:        https://repo1.maven.org/maven2/org/ow2/asm/asm-commons/%{version}/asm-commons-%{version}.pom
+Source5:        https://repo1.maven.org/maven2/org/ow2/asm/asm-test/%{version}/asm-test-%{version}.pom
+Source6:        https://repo1.maven.org/maven2/org/ow2/asm/asm-tree/%{version}/asm-tree-%{version}.pom
+Source7:        https://repo1.maven.org/maven2/org/ow2/asm/asm-util/%{version}/asm-util-%{version}.pom
+Source9:        generate-tarball.sh
```

Comment:
Is this source used anywhere in the spec? In general, we don't include the source generation tarball in the spec as it is mostly used for the upload of the source tar itself.

Reply:
Good point. Removed generate-tarball.sh from source.